scripts/release/debian/build.sh

```diff
@@ -21,8 +21,17 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # Update APT packages
 apt-get update
 # uuid-dev is used to build _uuid module: https://github.com/python/cpython/pull/3796
-apt-get install -y libssl-dev libffi-dev python3-dev debhelper zlib1g-dev uuid-dev
-apt-get install -y wget
+apt-get install -y libssl-dev libffi-dev python3-dev zlib1g-dev uuid-dev wget
+
+# In Ubuntu 18.04, debhelper 11.1.6 has bug which makes it fail to dpkg-buildpackage. Use backport version instead.
+# https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=897569
+if cat /etc/lsb-release | grep 18.04
+then
+  apt-get install -y debhelper -t bionic-backports
```

Comment:
Any reference for installing `bionic-backports` as `-t, --target-release, --default-release`? https://manpages.ubuntu.com/manpages/focal/man8/apt-get.8.html

Reply:
`debhelper` 13.6 is availabel in this repo, so I have to install from it.
https://packages.ubuntu.com/bionic-backports/devel/debhelper